doc/api/crypto.md

```diff
@@ -1121,6 +1121,8 @@ changes:
 * `privateKey` {string | Object}
   - `key` {string}
   - `passphrase` {string}
+  - `padding` {integer}
```

Comment:
I know it seems sensible to keep `key` and `passphrase` near each other, but I wonder if we would be best to alphabetize the options always. Especially in `crypto`, the options lists can get long and having a clear and obvious ordering is helpful when you're looking for that one option you care about.